### PR TITLE
Feature mes 4534 test requirements cat d

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
Added new multi legal requirement competency. This has two ticks instead of 1 for a competency.
The 1st press results in 1 tick. 2nd Press in 2 ticks. 3rd Press resets to 0 ticks.

Converted NS to a muti legal requirement, and added BS (bus stop) as a multi legal requirement.

Added tests.

## Checklist

- [X] PR title includes the JIRA ticket number
- [X] Branch is rebased against the latest develop
- [X] Code has been tested manually
- [X] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
NOTE: Screen shots to follow, once I can get someone to deploy to an ipad on my behalf and take a couple of screen shots.

### CODE REVIEW

**Review by TomBillingtonUK — APPROVED** [review]
Superb

File: src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(MultiLegalRequirementComponent);
+    component = fixture.componentInstance;
+    store$ = TestBed.get(Store);
+    storeDispatchSpy = spyOn(store$, 'dispatch');
+  }));
+
+  describe('Class', () => {
+    it('should dispatch a TOGGLE_NORMAL_START_1 action if neither requirement is  ticked', () => {
```
Comment (mgoldson) [inline src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts]: Looks like a double space before 'ticked'

File: src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    component.requirement1Ticked = true;
+    component.legalRequirement2 = LegalRequirements.normalStart2;
+    component.requirement2Ticked = true;
+    component.toggleLegalRequirement();
+    expect(storeDispatchSpy).toHaveBeenCalledTimes(2);
+    expect(storeDispatchSpy).toHaveBeenCalledWith(new ToggleLegalRequirement(LegalRequirements.normalStart1));
+    expect(storeDispatchSpy).toHaveBeenCalledWith(new ToggleLegalRequirement(LegalRequirements.normalStart2));
+  });
+
+  it('should dispatch a TOGGLE_BUS_STOP_1 action if neither requirement is  ticked', () => {
```
Comment (mgoldson) [inline src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts]: Looks like a double space before 'ticked'

**Review by mgoldson — APPROVED** [review]
Awesome work!

**Review by appleabs — APPROVED** [review]

File: src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts (hunk 2 above)
Comment (rossfauldsbjss) [reply src/pages/test-report/components/multi-legal-requirement/__tests__/multi-legal-requirement.spec.ts]: removed space

**Pull request merged**